cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -365,7 +365,7 @@ runSetup verbosity setup args0 = do
 -- verbosity applies to ALL commands.
 verbosityHack :: Version -> [String] -> [String]
 verbosityHack ver args0
-    | ver >= mkVersion [1,25] = args0
+    | ver >= mkVersion [2,1]  = args0
```

Comment:
@23Skidoo  Done, `filterConfigureFlags` adapted as per 51d3a2d48ae5b6dda56f3fdd29b37ff4cc6cd1d5 - everytime I have to touch `filterConfigureFlags` I get confused... :-)